susemanager/src/mgr_bootstrap_data.py

```diff
@@ -731,6 +731,62 @@
     "salt-minion"
 ]
 
+PKGLISTDEBIAN11 = [
+    "dctrl-tools",
+    "debconf-utils",
+    "dirmngr",
+    "distro-info-data",
+    "gnupg",
+    "gnupg-l10n",
+    "gnupg-utils",
+    "gpg",
+    "gpg-agent",
+    "gpgconf",
+    "gpgsm",
+    "gpg-wks-client",
+    "gpg-wks-server",
+    "iso-codes",
+    "libassuan0",
+    "libksba8",
+    "libldap-2.4-2",
+    "libldap-common",
+    "libnorm1",
+    "libnpth0",
+    "libpgm-5.2-0",
+    "libsasl2-2",
+    "libsasl2-modules",
+    "libsasl2-modules-db",
+    "libsodium23",
+    "libyaml-0-2",
+    "libzmq5",
+    "lsb-release",
+    "pinentry-curses",
+    "python3-apt",
+    "python3-certifi",
+    "python3-chardet",
+    "python3-croniter",
+    "python3-crypto",
```

Comment:
No such package on Debian 11.

Reply:
It seems we'll need to do a big cleanup of this for Debian 11.

Since Debian 11 provides a newer salt version what we provide, this means Debian 11 *must* use the Salt bundle when it's available.

For now I will test with the bundle, to get a feeling of how things are doing, but the Debian 11 will need to wait a bit more until the bundle can be released at least as Tech Preview.